Source/cmake/WebKitCompilerFlags.cmake

```diff
@@ -128,13 +128,17 @@ if (COMPILER_IS_GCC_OR_CLANG)
     endif ()
 
     # Warnings to be enabled
-    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-Wcast-align
-                                         -Wformat-security
+    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-Wformat-security
                                          -Wmissing-format-attribute
                                          -Wpointer-arith
                                          -Wundef
                                          -Wwrite-strings)
 
+    # Warnings to be enabled only on 64-bits
+    if (CMAKE_SIZEOF_VOID_P EQUAL 8)
+        WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-Wcast-align)
+    endif ()
```

Comment:
I'm not sure if this warning is really useful at all if we're going to disable it for 32-bit systems. x86 does not care about unaligned accesses, so this warning is really there for ARM, right? Maybe just remove it? The list of extra warnings that we want above and beyond those provided by -Wall should ideally be kept short, after all.

Reply:
Good point.
I disabled it because warnings are now fatal by default and there are a few warnings of this which I have no much idea how to fix. But I see now how disabling this warning is not the best idea. 

I guess it will be a better idea to make warnings non-fatal by default for 32-bits maybe.